roles/platform/tasks/initialize_setup_aws.yml

```diff
@@ -78,6 +78,7 @@
 - name: Handle AWS Public and Private VPC Subnets if not defined
   when: not plat__aws_public_subnet_ids or not plat__aws_private_subnet_ids
   block:
+    # Search using a wildcard on the subnet Name tag {{ plat__namespace }}*
```

Comment:
I'm confused by this comment

Reply:
Sorry, a hold-over from an older version of my code. Removed the comment now.